tnoodlejs/readme.txt

```diff
@@ -0,0 +1,22 @@
+Note: This is the documentation for USING tnoodle in JavaScript. This is NOT on EDITING tnoodle
+in JavaScript.
+If you want a working example of tnoodle, then you can open demo.html (in the current directory) 
+in your browser, and you should see a couple of puzzle icons. When you click one of them, it 
+generates a scramble for that puzzle.
+
+The tnoodle.js file in the root directory (1 level above this) is what you need. So, the function
+for generating scrambles can be called like this:
+
+function puzzlesLoaded(puzzles) {
+  puzzles['333'].generateScramble()
+          ^  This is where you should put the puzzle type.
```

Comment:
Can you make this a javascript comment? Right now, if someone copy pasted it, it would not run.

Reply:
@jfly oh, sure! I did not see that!